frontend/keyboardNav.js

```diff
@@ -104,15 +104,15 @@ function getRootSelection(dest, store, id) {
     return store.skipWrapper(roots.get(ix + 1));
   }
   return null;
-	}
+}
 
 function getNewSelection(dest: Dest, store: Store): ?ElementID {
   var id = store.selected;
   if (!id) {
     return undefined;
   }
   var node = store.get(id);
-  var pid = store.skipWrapper(store.getParent(id), true);
```

Comment:
It was actually what I was doing before, but failed to copy it over to my PR (it was working on the example I showed you locally on my Mac) in the compiled bundle. N00b error.

We don't want to skip the wrapper in case we're needing to traverse an actual wrapper. This breaks it.